core/src/Cache/CacheFallbackOperations.cs

```diff
@@ -74,7 +81,9 @@ internal class CacheFallbackOperations
         {
             if (rtItem == null)
             {
-                CoreLoggerBase.Default.Info("rtItem is null. Skipping MSAL RT cache write");
+                string msg = "rtItem is null. Skipping MSAL RT cache write";
```

Comment:
>rtItem is null.  [](start = 30, length = 16)

No refresh token available. (the current text is not something imo that a developer would understand unless they look into our code)

Reply:
yes...definitely. Good catch. wasn't paying much attention to what was actually in the message. thx

---
In reply to: [206365110](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1154#discussion_r206365110) [](ancestors = 206365110)